continuous_integration/environment-3.9.yaml

```diff
@@ -30,8 +30,7 @@ dependencies:
   - coverage
   - jsonschema
   # other -- IO
-  # Not available for Python 3.9 on conda-forge yet
-  # - bcolz
+  - bcolz
```

Comment:
This change is causing conda solve issues in CI since `bcolz` still isn't available on Python 3.9 yet (and I don't know if it ever will be). Since `bcolz` is [no longer maintained](https://github.com/Blosc/bcolz#unmaintained-package-notice), we might consider deprecating support for it in Dask.

Reply:
oh strange. I could have sworn I checked that it was available.

Reply:
No worries -- FWIW I opened up https://github.com/dask/dask/issues/8330 to see if there's interest in deprecating `from_bcolz`